### PR TITLE
Physics

### PR DESCRIPTION
**Pull request by randomPoison**
- Makes the lifter a separate rigid body to better take advantage of Unity's physics constraints.

### CODE REVIEW

File: Assets/Scripts/Actors/ActorPhysics.cs [hunk 1]
```diff
@@ -338,7 +347,10 @@ public void DoJump()
 	 */
 	public void AirMovement( Vector3 inputVec, bool forceUp = false )
 	{
+		_rigidbody.useGravity = true;
+
 		FollowBumper();
+		//FollowLifter();
```
Comment (brickman1444) [inline Assets/Scripts/Actors/ActorPhysics.cs]: Should this be here?

File: Assets/Scripts/Player/PlayerControls.cs [hunk 2]
```diff
@@ -111,6 +116,7 @@ public override void Enter()
 
 		public override void Update()
 		{
+			player.animator.SetBool( "isInAir", true );
```
Comment (brickman1444) [inline Assets/Scripts/Player/PlayerControls.cs]: Why is this here instead of the enter method?

**Comment by brickman1444** [comment]
done reviewing

File: Assets/Scripts/Actors/ActorPhysics.cs (hunk 1 above)
Comment (randomPoison) [inline Assets/Scripts/Actors/ActorPhysics.cs]: Nope, done.

File: Assets/Scripts/Player/PlayerControls.cs (hunk 2 above)
Comment (randomPoison) [inline Assets/Scripts/Player/PlayerControls.cs]: I'll change this to an invoke.
